site/package.json

```
@@ -16,6 +16,7 @@
     "bootstrap": "4.6.0",
     "chosen-js": "1.8.7",
     "codemirror": "5.62.0",
+    "codemirror-spell-checker": "^1.1.2",
```

We don't use ranges for dependencies (though there are some that do mistakenly at the moment). This is because Submitty is an app and not a library.

```suggestion
    "codemirror-spell-checker": "1.1.2",
```